homework/salesdata.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+import pandas as pd
+import os
+import psycopg2
+
+os.chdir('/home/bcohen/Documents/BGSE/Classes/1_Warehousing/warehousing-operational-databases')
+os.listdir('homework/extracts/')
+
+employees = pd.read_csv('homework/extracts/employees.csv')
+orders = pd.read_csv('homework/extracts/orders.csv')
+products = pd.read_csv('homework/extracts/products.csv')
+
+[print(c) for c in employees.columns if c in orders.columns] # city, state, country
+[print(c) for c in employees.columns if c in products.columns] # none!
+[print(c) for c in orders.columns if c in products.columns] # product_code
+
+employees.shape
+employees.drop_duplicates().shape
+
+len(employees.employee_number.drop_duplicates())
+
+products.shape
+products.drop_duplicates().shape
+
+orders.shape
+orders.drop_duplicates().shape
+
+orders['order_item_id'] = np.arange(len(orders))
```

Comment:
This is a good thing to give to your database! Postgres has the type SERIAL which creates an auto-incrementing column and is common for this use case. That way this script could be run more than once to load new data!